src/test/scala/org/camunda/feel/impl/builtin/BuiltinContextFunctionsTest.scala

```diff
@@ -54,13 +54,16 @@ class BuiltinContextFunctionsTest
   }
 
   "A get value function" should "return the value" in {
-
     eval(""" get value({foo: 123}, "foo") """) should be(ValNumber(123))
   }
 
-  it should "return null if not contains" in {
+  "A get value function" should "return the value when arguments are named 'm' and 'key'" in {
```

Comment:
:x: We should group test cases by using it. The test case will be in the same group as the previous test case (i.e. "A get value function").

```suggestion
  it should "return the value when arguments are named 'm' and 'key'" in {
```

Reply:
:x: We should use the old parameter name in the previous test case. So, we can verify that the previous behavior still works.